components/browser/engine-gecko-beta/src/main/java/mozilla/components/browser/engine/gecko/GeckoEngineSession.kt

```diff
@@ -180,6 +181,7 @@ class GeckoEngineSession(
     /**
      * See [EngineSession.findNext]
      */
+    @SuppressLint("WrongConstant")
```

Comment:
Can you add a comment explaining why this falsely triggers "WrongConstant"?

Reply:
Sure, it was the linter that complains since there are IntDef values for `@FinderFindFlags` that don't include the default 0 value. I'll add that in a comment by the annotation and I've filed a bug for GV ([bug 1489528][0]).

[0]: https://bugzilla.mozilla.org/show_bug.cgi?id=1489528